### ISSUE TITLE
Odd 15 HR Recruitment Send Interview Error

### ISSUE THREAD

**Issue by younismahsud** [odoo/odoo#110237]
Impacted versions: 15.0

Steps to reproduce:

When clicking on Send Interview button on the Applicant form, a wizard is opened up. Then select multiple recipients and click Send button. You will get the following error.

```
RPC_ERROR
Odoo Server Error
Traceback (most recent call last):
  File "/data/build/odoo/odoo/addons/base/models/ir_http.py", line 237, in _dispatch
    result = request.dispatch()
  File "/data/build/odoo/odoo/http.py", line 687, in dispatch
    result = self._call_function(**self.params)
  File "/data/build/odoo/odoo/http.py", line 359, in _call_function
    return checked_call(self.db, *args, **kwargs)
  File "/data/build/odoo/odoo/service/model.py", line 94, in wrapper
    return f(dbname, *args, **kwargs)
  File "/data/build/odoo/odoo/http.py", line 348, in checked_call
    result = self.endpoint(*a, **kw)
  File "/data/build/odoo/odoo/http.py", line 916, in __call__
    return self.method(*args, **kw)
  File "/data/build/odoo/odoo/http.py", line 535, in response_wrap
    response = f(*args, **kw)
  File "/data/build/odoo/addons/web/controllers/main.py", line 1346, in call_button
    action = self._call_kw(model, method, args, kwargs)
  File "/data/build/odoo/addons/web/controllers/main.py", line 1334, in _call_kw
    return call_kw(request.env[model], method, args, kwargs)
  File "/data/build/odoo/odoo/api.py", line 464, in call_kw
    result = _call_kw_multi(method, model, args, kwargs)
  File "/data/build/odoo/odoo/api.py", line 451, in _call_kw_multi
    result = method(recs, *args, **kwargs)
  File "/data/build/odoo/addons/hr_recruitment_survey/models/survey_invite.py", line 28, in action_invite
    return super().action_invite()
  File "/data/build/odoo/addons/survey/wizard/survey_invite.py", line 277, in action_invite
    answers = self._prepare_answers(valid_partners, valid_emails)
  File "/data/build/odoo/addons/survey/wizard/survey_invite.py", line 200, in _prepare_answers
    answers |= self.survey_id._create_answer(partner=new_partner, check_attempts=False, **self._get_answers_values())
  File "/data/build/odoo/addons/survey/models/survey_survey.py", line 397, in _create_answer
    user_inputs += user_inputs.create(answer_vals)
  File "<decorator-gen-957>", line 2, in create
  File "/data/build/odoo/odoo/api.py", line 412, in _model_create_multi
    return create(self, [arg])
  File "/data/build/odoo/addons/website_slides_survey/models/survey_user.py", line 18, in create
    records = super(SurveyUserInput, self).create(vals_list)
  File "<decorator-gen-232>", line 2, in create
  File "/data/build/odoo/odoo/api.py", line 413, in _model_create_multi
    return create(self, arg)
  File "/data/build/odoo/addons/survey/models/survey_user_input.py", line 159, in create
    return super(SurveyUserInput, self).create(vals_list)
  File "<decorator-gen-67>", line 2, in create
  File "/data/build/odoo/odoo/api.py", line 413, in _model_create_multi
    return create(self, arg)
  File "/data/build/odoo/odoo/addons/base/models/ir_fields.py", line 613, in create
    recs = super().create(vals_list)
  File "<decorator-gen-13>", line 2, in create
  File "/data/build/odoo/odoo/api.py", line 413, in _model_create_multi
    return create(self, arg)
  File "/data/build/odoo/odoo/models.py", line 4016, in create
    vals = self._add_missing_default_values(vals)
  File "/data/build/odoo/odoo/models.py", line 1956, in _add_missing_default_values
    defaults = self.default_get(list(missing_defaults))
  File "/data/build/odoo/odoo/models.py", line 1429, in default_get
    value = field.convert_to_cache(value, self, validate=False)
  File "/data/build/odoo/odoo/fields.py", line 3197, in convert_to_cache
    raise ValueError("Wrong value for %s: %s" % (self, value))
Exception

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/data/build/odoo/odoo/http.py", line 643, in _handle_exception
    return super(JsonRequest, self)._handle_exception(exception)
  File "/data/build/odoo/odoo/http.py", line 301, in _handle_exception
    raise exception.with_traceback(None) from new_cause
ValueError: Wrong value for survey.user_input.applicant_id: 10
```

Current behavior:

I can send the Interview request to one recipient at a time but when I select two or more then I get an error.

Expected behavior:

It should send the interview request whether I select one recipient or more.

Video/Screenshot link (optional):

Support ticket number submitted via odoo.com/help (optional):

**Comment by mehjabinfarsana** [comment]
fixed by: https://github.com/odoo/odoo/commit/9c936f4afbc5df2a94dc0f3e230dc2892e151dba

**Comment by younismahsud** [comment]
> fixed by: [9c936f4](https://github.com/odoo/odoo/commit/9c936f4afbc5df2a94dc0f3e230dc2892e151dba)

Thank you.
